hs/WebParsing/UTSCParser.hs

```diff
@@ -6,7 +6,7 @@ import Text.HTML.TagSoup.Match
 import Database.Persist.Sqlite
 import qualified Data.Text as T
 import Database.Tables as Tables
-import Database.JsonParser as JsonParser
+import Database.CourseInsertion as CourseInsertion
```

Comment:
This was was funny because nothing else in the file changed. Seems like you should just remove this import.